ports/stm32/Makefile

```diff
@@ -93,6 +96,7 @@ endif
 
 LDFLAGS = -nostdlib -L $(LD_DIR) $(addprefix -T,$(LD_FILES)) -Map=$(@:.elf=.map) --cref
 LIBS = $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
+LIBS += cyw43/libcyw43.a
```

Comment:
Any reason why this is a library and not as source?

Reply:
Most of it is a binary blob that's downloaded to the CYW4343.  The rest is a small bit of glue that talks to the CYW4343 through the SDMMC bus, and needs to be released as a binary to comply with the Cypress licensing.

Reply:
First I  glad we have the binary version at least. Thanks to the authors.  I would be nice to have the glue logic as source. Cypress has released an apace licensed version of WWD driver https://github.com/cypress/wwd. This has the glue logic.

Reply:
Thanks for the link to the Cypress repo, that's really great news that they have released it like that!  So I think then we can make this glue logic open source.  I will look into it.

Reply:
Hi Damien George,
I am trying flash the mpy on PYBD-SF3-W4F2. I got the build successful and generated the *.dfu. But unable to get the board into DFU mode for flashing the image. Could please provide me a pointer on this?

Thanks for the help,
KVVD

Reply:
@ Damien George,
Continuing my previous post. Steps followed to get the board into DFU mode:
1. connect 3v3 pin with boot0 pin.
2. Press reset button on the board
3. Check if the device is listed using "dfu-tool watch" or "dfu-util -l".
    But no luck. Empty list is printed on the screen.

Reply:
@kvvd-packetp

http://micropython.org/download has the instructions for getting into the bootloader for the pybd.

The background here is that pybd uses its own bootloader (mboot) which has more features than the built-in ST bootloader (mainly it has the ability to write to external flash, so that the dfu file can flash both the on-chip flash as well as the external qspiflash). It's triggered by the process described above.

The likely reason the ST bootloader (which is what you're getting with the boot0 pin) isn't working for you is that it uses the other USB interface (which is accessible on the WBUS-DIP28 board), but either way you won't be able to flash a .dfu file built for the pybd using it.

Reply:
Thanks a lot for the update. I am able to get the board into dfu mode.

Reply:
> Any reason why this is a library and not as source?

@dpgeorge Any plans on releasing this as open source. If there is a project plan for this, I would love to contribute!

Reply:
@vs-pp
This just happened last week -- the driver is now dual-licensed (free for non-commercial or with Raspberry Pi devices, commercial licensing on request). See https://github.com/georgerobotics/cyw43-driver